lib/matplotlib/patches.py

```diff
@@ -4256,67 +4277,23 @@ def draw(self, renderer):
         if not self.get_visible():
             return
 
-        renderer.open_group('patch', self.get_gid())
-        gc = renderer.new_gc()
-
-        gc.set_foreground(self._edgecolor, isRGBA=True)
-
-        lw = self._linewidth
-        if self._edgecolor[3] == 0:
-            lw = 0
-        gc.set_linewidth(lw)
-        gc.set_dashes(self._dashoffset, self._dashes)
```

Comment:
The context uses `gc.set_dashes(0, self._dashes)` instead.

Reply:
Indeed, I had missed that.  I restored the old behavior using setattr_cm (in Patch.draw), leaving a comment similar to the one for the capstyle in FancyArrowPatch.